cmd/run.go

```diff
@@ -241,11 +241,11 @@ func buildGraph(fileSystem *fs.FileSystem, modelFilters []string) *fs.Graph {
 				model := fileSystem.Model(modelFilter)
 				if model == nil {
 					if FailOnNotFound {
-						pb.Stop()
```

Comment:
The pb.Stop needs to be first, as the progress bar go routnies are muitating the stdout / stderr interface